components/RoundButton.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <a :href="url">
+    <v-btn rounded>{{ name }}</v-btn>
+  </a>
```

Comment:
```suggestion
  <v-btn rounded :href="url">{{ name }}</v-btn>
```
v-btnにhref属性があることを忘れていました....
こちらもsuggestionを取り込んでもらえばと思います。